build.sc

```diff
@@ -28,7 +24,7 @@ trait SourcecodeModule extends PublishModule with MimaCheck {
   // Temporary until the next version of Mima gets released with
   // https://github.com/lightbend/mima/issues/693 included in the release.
   def mimaPreviousArtifacts =
-    if(isScala3(crossScalaVersion)) Agg.empty[Dep] else super.mimaPreviousArtifacts()
+    if(crossScalaVersion.startsWith("3.")) Agg.empty[Dep] else super.mimaPreviousArtifacts()
```

Comment:
`isScala3` still exists but is inside `ZincWorkerUtil` now I believe

Reply:
fixed